packages/rpc-graphql/src/resolvers/instruction.ts

```diff
@@ -882,6 +888,9 @@ export const instructionResolvers = {
                     if (jsonParsedConfigs.instructionType === 'initializeConfidentialTransferFeeConfig') {
                         return 'SplTokenInitializeConfidentialTransferFeeConfig';
                     }
+                    if (jsonParsedConfigs.instructionType === 'initializeTokenGroup') {
+                        return 'SplTokenInitializeTokenGroup';
+                    }
```

Comment:
This is still in the right spot, since for now the RPC is only going to parse this for Token-2022.